modules/pxp-module-puppet

```diff
@@ -233,6 +242,9 @@ end
 
 def get_action_input(args)
   action_input = args["input"]
+  unless action_input.is_a?(Hash)
+    raise ProcessingError.new(Errors::InvalidJson, "The json received on STDIN did not contain a valid 'input' key.")
```

Comment:
Maybe add `args.to_s` to the error message so we could get some hint about what is wrong with args?